fix: edit review accessibility (#228)
plentymarkets/feedback-plugin: meta/documents/changelog_en.md

# Release Notes for Feedback

## v5.0.2 Next release <a href="https://github.com/plentymarkets/feedback-plugin/compare/5.0.1...5.0.2" target="_blank" rel="noopener"><b>Overview of all changes</b></a>

### Fixed

> - A problem with the accessibility of the review form has been fixed.

Reviewer: ```suggestion
- The accessibility of the review form has been improved.
```
